doc/source/release_notes.rst

```diff
@@ -55,6 +41,121 @@ API Changes
       :func:`ophyd.setup_ophyd`, which is still available as a top-level
       import.
 
+* For complex devices, the same child device may be used in both ``read_attrs``
+  and ``read_configuration``.  Putting the read values into the configuration
+  is generically not correct. For example, the mean_value of a stats plugin for
+  Area Detector should be in the ``read()`` but not in the result of
+  ``read_configuration()``. At the bottom, Signals fall back to ``read()`` for
+  their read_configuration implementation.
+* The area detector 'EnableCallbacks' signal is set using its integer
+  representation instead of its enum string. The string representation was
+  changed on the NDPluginBase.template file in upstream Area Detector. The int
+  value is stable (we hope).
+* Low-level changes related to integration between ophyd's area detector code
+  and databroker/filestore:
+
+    * Ophyd's optional dependency on filestore, which is now a deprecated
+      package, has been replaced by an optional dependency on databroker. In
+      area detector classes, the keyword argument and attribute ``fs`` has been
+      changed to ``reg``, short for "registry".
+    * The ``FileStoreBulkWrite`` mixin classes have been removed. Now that the
+      Registry is generating the datum UIDs the 'stash, emit on read, and then
+      insert on unstage' is no longer possible.  This means we will never let a
+      datum_id which is not in a Registry out into the EventSources.  This
+      change is driven by the need to support column based backends from Assets.
+    * The method ``generate_datum`` on area detector file plugins requires an
+      additional argument, ``datum_kwargs``.
+
+Enhancements
+************
+
+* Many devices picked up a new ``hints`` property. Its goal is to highlight the
+  most interesting or important fields---often a small subset of all the fields
+  that are read---in support of automated visualization and processing. It does
+  not affect what is read or recorded; nothing is permanently altered or lost
+  if the hints are incorrect. The content of hints may be changed in future
+  releases, as this feature is experimental. Movable devices return a
+  dictionary with the key ``dimensions`` mapped to a list of field names,
```

Comment:
I thought 'dimensions' was a plan level thing?  I don't think anything in ophyd actually returns a anything but `{'fields': [...]}`